iqm5q.py

```diff
@@ -21,6 +21,7 @@
 TWPA_ADDRESS = "192.168.0.35"
 
 N_QUBITS = 5
+KERNEL_FOLDER = "qibolab_platforms_qrc/iqm5q_kernels/"
```

Comment:
```suggestion
KERNEL_FOLDER =pathlib.Path(__file__).parent/ "iqm5q_kernels"
```
I think in this way it should be better.